### ISSUE TITLE
can not share password profile (chrome 96/firefox 95)

### ISSUE THREAD

**Issue by NateScarlet** [lesspass/lesspass#678]
```
ReferenceError: copy is not defined
    sharePasswordProfile PasswordGenerator.vue:285
    click PasswordGenerator.vue:1
```

![图片](https://user-images.githubusercontent.com/21083014/146395520-9a561ef8-9783-4213-b5d0-ffba38501e1c.png)

**Comment by guillaumevincent** [comment]
Hello,
where do you have this issue ?
I don't have it on the web site or on the web extension.

Thank you for reporting this

**Comment by NateScarlet** [comment]
both https://www.lesspass.com/#/ and firefox/chrome extension

![图片](https://user-images.githubusercontent.com/21083014/146408689-8750b6be-fe80-462a-aca3-85a6d04a2dd5.png)

**Comment by guillaumevincent** [comment]
Oh missed the fact it's in the share password feature
I will fix this

**Comment by guillaumevincent** [comment]
Will be fixed in version 9.5.0 on the different stores
I will update the web site soon

**Comment by guillaumevincent** [comment]
Closing, fixed in production
